dashboard/app/assets/stylesheets/application.scss

```diff
@@ -983,7 +983,7 @@ $default-modal-width: 640px;
 .header_right {
   position: absolute;
   float: left;
-  right: 25px;
+  right: 16px;
```

Comment:
nit: I think you can get rid of `float: left;` since it should have no effect on an absolutely positioned element.